### ISSUE TITLE
no module named "requests" error

### ISSUE THREAD

**Issue by whchi** [blockchain-certificates/cert-issuer#71]
Hi, I'm trying to install this, but when I run `docker build` command, it's always return errors below: 
```
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-o_d_2zxh/pyld/setup.py", line 17, in <module>
        from pyld import jsonld
      File "/tmp/pip-build-o_d_2zxh/pyld/lib/pyld/__init__.py", line 2, in <module>
        from . import jsonld
      File "/tmp/pip-build-o_d_2zxh/pyld/lib/pyld/jsonld.py", line 19, in <module>
        import requests
    ImportError: No module named 'requests'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-o_d_2zxh/pyld/
```
unless I append 'requests' in Dockerfile's pip3 install, can you help me with that ?

**Comment by kimdhamilton** [comment]
I am able to repro; it appears to be an issue in pyld with python 3.6. My best guess is that relative imports (which it's using in __init__.py) is causing problems. 

I have a minimal repro with requirements.txt as follows:
```
pyld>=0.8.0
requests>=2.18.4
```
Creating a python 3.6 virtualenv causes this to fail with:

```
Collecting pyld>=0.8.0 (from -r requirements.txt (line 1))
  Using cached PyLD-0.8.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/y3/91nv4d6n76bfp7zjtnx40wrr0000gn/T/pip-build-bteip65y/pyld/setup.py", line 17, in <module>
        from pyld import jsonld
      File "/private/var/folders/y3/91nv4d6n76bfp7zjtnx40wrr0000gn/T/pip-build-bteip65y/pyld/lib/pyld/__init__.py", line 2, in <module>
        from . import jsonld
      File "/private/var/folders/y3/91nv4d6n76bfp7zjtnx40wrr0000gn/T/pip-build-bteip65y/pyld/lib/pyld/jsonld.py", line 19, in <module>
        import requests
    ModuleNotFoundError: No module named 'requests'
```

I note that it passes in 2.7 and 3.4 (the other versions I tested).

I'm going to see if there is a way to work around this for python 3.6

**Comment by kimdhamilton** [comment]
here is a minimal repro in travis: https://travis-ci.org/kimdhamilton/pyld-test/jobs/291439728

**Comment by kimdhamilton** [comment]
and actually, I think my note on 2.7 and 3.4 is specific to my environment. It seems to fail in all environments when I run in travis:
https://github.com/kimdhamilton/pyld-test

**Comment by whchi** [comment]
Hi, I've tried some possible ways and finds out that this issue should belongs the library pyld: 
https://github.com/digitalbazaar/pyld

I run it's install command `pip install PyLD` with python 2.7 / python 3+, came out the same error, I'll reply to them.

Thanks for your help!

**Comment by kimdhamilton** [comment]
Issue has already been filed: https://github.com/digitalbazaar/pyld/issues/68

I added details

**Comment by davidlehn** [comment]
pyld 0.8.1 should fix the requests installation issue.

**Comment by kimdhamilton** [comment]
Fixed in pyld. This project build is fixed as of f8147d4de05dc0f0764c88a7c659e5739c0a77e4
